firmware/gameport-adapter/gameport-adapter.ino

```diff
@@ -13,6 +14,12 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+/// DEBUG information: Debugging is turned off by default
+/// Comment the "NDEBUG" line for direct message output to the Serial Monitor.
+/// Since the Arduino Micro model directly shares Serial with native USB interface
+/// the underlying operating system will no longer recognize the USB device!
+#define NDEBUG
```

Comment:
Ah, I see the no debug flag is now defined here. May be it's a good idea to move it into the Utilities.h, like you did before? I liked it there because, I could instantly see, what it means. As I said once before, I really don't like macros in C++, but if I have to define one, I try to keep them as bundled as possible.

Reply:
All related NDEBUG macro code was moved to Utilities.h now